### PR TITLE
[RDBMS] `az mysql flexible-server server-logs`: Add server logs for MySQL Flexible Server

### PR DESCRIPTION
**Pull request by alanenriqueo**
**Related command**
`az mysql flexible-server server-logs download`
`az mysql flexible-server server-logs list`

**Description**
- Add server logs for MySQL Flexible Server

**Testing Guide**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
RDBMS

File: src/azure-cli/requirements.py3.Darwin.txt [hunk 1]
```diff
@@ -33,7 +33,7 @@ azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==9.1.0
 azure-mgmt-containerregistry==8.2.0
 azure-mgmt-containerservice==19.1.0
-azure-mgmt-core==1.3.0
+azure-mgmt-core==1.3.1
```
Comment (DaeunYim) [inline src/azure-cli/requirements.py3.Darwin.txt:36]: Any specific reason this had to change?

Comment (alanenriqueo) [reply src/azure-cli/requirements.py3.Darwin.txt:36]: In package `azure-mgmt-rdbms==10.2.0b1` we have the requirement: `Requires-Dist: azure-mgmt-core (<2.0.0,>=1.3.1)`

File: src/azure-cli/azure/cli/command_modules/rdbms/_params.py [hunk 2]
```diff
@@ -604,6 +604,18 @@ def _flexible_server_params(command_group):
...
+        # logs
+        if command_group == 'mysql':
+            with self.argument_context('{} flexible-server server-logs download'.format(command_group)) as c:
+                c.argument('server_name', id_part='name', options_list=['--server-name', '-s'], arg_type=server_name_arg_type, help='Name of the Server.')
+                c.argument('file_name', options_list=['--name', '-n'], nargs='+', help='Space-separated list of log filenames on the server to download.')
+
+            with self.argument_context('{} flexible-server server-logs list'.format(command_group)) as c:
+                c.argument('server_name', id_part='name', options_list=['--server-name', '-s'], arg_type=server_name_arg_type, help='Name of the Server.')
+                c.argument('filename_contains', help='The pattern that file name should match.')
+                c.argument('file_last_written', type=int, help='Integer in hours to indicate file last modify time, default value is 72.')
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/rdbms/_params.py:616]:
```suggestion
                c.argument('file_last_written', type=int, help='Integer in hours to indicate file last modify time', default=72)
```

Comment (alanenriqueo) [reply src/azure-cli/azure/cli/command_modules/rdbms/_params.py]: Done, thanks.

**Pull request merged**